gems/aws-sdk-s3/spec/object/multipart_copy_spec.rb

```diff
@@ -257,8 +257,7 @@ module S3
             allow(client).to receive(:head_object).with({
               bucket: 'source-bucket',
               key: 'source key'
-            }).and_return(Types::HeadObjectOutput.new(
-              content_length: size, content_type: 'ContentType'))
+            }).and_return(Types::HeadObjectOutput.new(content_length: size, content_type: 'ContentType'))
```

Comment:
Yuck.. I'm not sure why this is here. Do you mind changing all of these to stub_data?